### ISSUE TITLE
Initialising with TelemetryClient

### ISSUE THREAD

**Issue by batizar** [serilog/serilog-sinks-applicationinsights#97]
There is an issue in the README for new V3 release `TelemetryConfiguration()` doesn't accept TelemetryClient.

https://docs.microsoft.com/en-us/dotnet/api/microsoft.applicationinsights.extensibility.telemetryconfiguration?view=azure-dotnet#constructors

```c#
var log = new LoggerConfiguration()
    .WriteTo
	.ApplicationInsights(new TelemetryConfiguration(_telemetryClient), TelemetryConverter.Events)
    .CreateLogger();
```

**Comment by mikanyg** [comment]
I would argue that the change to remove overloads for injecting an instance of telemetry client is wrong. The reason being that we cannot call Flush on the instance that is created by the sink. V2 of the sink had the option to inject a TelemetryClient where we could control the lifecycle.

So the current documentation for Flush is showcasing something that cannot be achieved in V3.

**Comment by aloneguid** [comment]
I will add it back. The reason for removing it was too many overloads that are never used/conflict with each other. At the moment everything in v3 is covered by unit tests, so I'll continue this approach and add it back.

**Comment by aloneguid** [comment]
fixed and released in 3.0.1, please reopen if any issues.
